src/main/resources/db/migration/V4__Add_column_papirSmRegistrering.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE manuelloppgave ADD COLUMN papir_sm_registrering JSONB NULL;
```

Comment:
Jeg ville kanskje ha gjort dette først i en egen PR, altså prodsette databaseendringen først og så prodsatt kodeendringene etterpå. På den måten slipper man å få problemer med å måtte rulle tilbake databasen manuelt hvis noe går galt og man må rulle tilbake :) Databaseendringen kan du jo gjøre i prod med en gang hvis du vil siden den er bakoverkompatibel :)

Reply:
Ah, smart. Jeg kan lage en egen PR for databaseendringene.

Reply:
Dette ligger i en egen PR her: https://github.com/navikt/smregistrering-backend/pull/3